modules/new_serverpod_auth/serverpod_auth_idp/serverpod_auth_idp_server/lib/src/providers/email/business/email_idp_config.dart

```diff
@@ -48,6 +48,15 @@ class EmailIDPConfig {
   /// The pepper used for hashing passwords and verification codes.
   final String secretHashPepper;
```

Comment:
Suggestion: Maybe add a comment here in the same fashion as the one on JWT?

```suggestion
  /// The pepper used for hashing passwords and verification codes.
  ///
  /// To rotate peppers without invalidating existing passwords, use [fallbackSecretHashPeppers].
  final String secretHashPepper;
```